src/app/components/info-card/info-card.tsx

```diff
@@ -20,29 +19,19 @@ export function InfoCard({ children, ...props }: InfoCardProps) {
 // InfoCardRow
 interface InfoCardRowProps {
   title: string;
-  value: React.ReactNode;
-  isAddressDisplayer?: boolean;
+  value: string | JSX.Element;
```

Comment:
Why not React.reactNode, that covers more input types I think?

Reply:
React.ReactNode is also boolean | null | undefined. I'd probably prefer more strict types here

Reply:
but I'm not much sure here